feature/nuvemshop-acessomensagem (#2646)
mercadopago/devsite-docs: guides/nuvemshop/integration.en.md

------------

> Ready! Installation was successful and you can now receive payments.

Reviewer: ```suggestion
Ready! Installation was successful and you can now receive payments.

```

## Revoke Mercado Pago access

> When you revoke Mercado Pago access from the integrations section of your account, the connection between your store and Nuvemshop is removed. This prevents the processing of new payments until the integration is restored.

Reviewer: ```suggestion
When you revoke Mercado Pago access from the integrations section of your account, the connection between your store and Tiendanube is removed. This prevents the processing of new payments until the integration is restored.
```

Reviewer: @22Neves, en español e inglés, el plugin se menciona como Tiendanube. En portugués como Nuvemshop ;)

> RED_MESSAGE
> 

> > This action cannot be undone automatically. We recommend revoking access only when you are certain that you want to disable Mercado Pago as a payment method in your store.

Reviewer: ```suggestion
> This action cannot be undone automatically. We recommend revoking access only when you are certain that you want to disable Mercado Pago as a payment method in your store.

```

> > This action cannot be undone automatically. We recommend revoking access only when you are certain that you want to disable Mercado Pago as a payment method in your store.

Reviewer: ```suggestion
> This action cannot be undone automatically. Revoke access only when you are certain that you want to disable Mercado Pago as a payment method in your store.
```

### After revoke permissions

> If access to Mercado Pago is revoked, your store will stop processing payments through the integrated methods on the platform. To reactivate payment processing, follow the steps below:

Reviewer: ```suggestion
If the access to Mercado Pago is revoked, your store will stop processing payments through the integrated methods on the platform. To reactivate payment processing, follow the steps below:
```

1. Go to the **integrations** section of your store in Nuvemshop and follow the Mercado Pago installation flow to authorize access again. 

> For more details, check the documentation [Integration setup] (/developers/pt/docs/nuvemshop/integration#bookmark_integration_configuration).

Reviewer: ```suggestion
For more details, check the documentation [Integration setup](/developers/pt/docs/nuvemshop/integration#bookmark_integration_configuration).
```

------------

Ready! Installation was successful and you can now receive payments.

## Revoke Mercado Pago access

When you revoke Mercado Pago access from the integrations section of your account, the connection between your store and Tiendanube is removed. This prevents the processing of new payments until the integration is restored.

> RED_MESSAGE
> 
> This action cannot be undone automatically. Revoke access only when you are certain that you want to disable Mercado Pago as a payment method in your store.

### After revoke permissions

If the access to Mercado Pago is revoked, your store will stop processing payments through the integrated methods on the platform. To reactivate payment processing, follow the steps below:

1. Go to the **integrations** section of your store in Tiendanube and follow the Mercado Pago installation flow to authorize access again. 
For more details, check the documentation [Integration setup](/developers/en/docs/nuvemshop/integration#bookmark_integration_configuration).

2. After completing the integration, review the configured payment options.
Check payment methods, installments, and preferences to ensure they meet your store's needs.

> Access restored!  Your store is ready to start processing payments again with Mercado Pago.

Reviewer 2: ```suggestion
Access restored! Your store is ready to start processing payments again with Mercado Pago.
```
